checks/tools/profiling_and_debugging/linaro-forge.py

```diff
@@ -0,0 +1,48 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class linaro_ddt(rfm.RunOnlyRegressionTest):
+    valid_systems = ['*']
+    valid_prog_environs = ['+ddt']
+    sourcesdir = None
```

Comment:
this is not necessary

Reply:
line 10 ? yes it is: I do not want/need to copy the src dir.